src/Compilers/CSharp/Portable/Symbols/EventSymbol.cs

```diff
@@ -117,7 +117,12 @@ public EventSymbol OverriddenEvent
             {
                 if (this.IsOverride)
                 {
-                    return (EventSymbol)OverriddenOrHiddenMembers.GetOverriddenMember();
+                    if (IsDefinition)
```

Comment:
We haven't run into any issues with hidden members yet. And it is fine to have multiple hidden members, or for a hidden member to hide an overridden member due to generic substitution. For the purpose of this API, however, we need to figure out the only truly overridden member, even if it is considered hidden due to generic substitution. To the best of my understanding, hidden members and an overridden member serve different purpose within compiler and it is fine for them to disagree within constructed types.